### PR TITLE
Selenium + selenide first tests added // 25.04.22

### PR DESCRIPTION
**Pull request by nataliashukalovich**
1) Firefox getTitle and openPage

image

2) Chrome getTitle and openPage

image

### CODE REVIEW

File: src/test/java/com/stormnet/ChromeMainPageTest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+public class ChromeMainPageTest {
+    private final String mainPage = "https://www.onliner.by/";
+    private final String catalogPage = "https://catalog.onliner.by/";
+
+    @Test
+    public void testOpenPage() {
+        open(mainPage);
+        open(catalogPage);
```
Comment (Kirill-Simonovich) [inline src/test/java/com/stormnet/ChromeMainPageTest.java]: open() use once per test

File: src/test/java/com/stormnet/ChromeMainPageTest.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        Selenide.closeWebDriver();
+    }
+
+    @Test
+    public void testPrintTitle() {
+        System.setProperty("webdriver.chrome.driver", "C:\\Users\\natalia.shukalovich\\test\\chromedriver.exe");
+        WebDriver driver = new ChromeDriver();
+        driver.get(mainPage);
```
Comment (Kirill-Simonovich) [inline src/test/java/com/stormnet/ChromeMainPageTest.java:29]: driver.get() use once per test

File: src/test/java/com/stormnet/ChromeMainPageTest.java (hunk 1 above)
Comment (Kirill-Simonovich) [reply src/test/java/com/stormnet/ChromeMainPageTest.java]: for redirect to another page 'open()' don't use, it's wrong

File: src/test/java/com/stormnet/FFMainPageTest.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+public class FFMainPageTest {
+    private final String mainPage = "https://belavia.by/";
+    private final String newsPage = "https://belavia.by/novosti/";
+
+    @Test
+    public void testOpenPage() {
+        System.setProperty("webdriver.gecko.driver", "C:\\Users\\natalia.shukalovich\\test\\geckodriver.exe");
+        WebDriver driver = new FirefoxDriver();
+        driver.get(mainPage);
+        driver.get(newsPage);
```
Comment (Kirill-Simonovich) [inline src/test/java/com/stormnet/FFMainPageTest.java]: this is work? How many windows are open?

**Review by Kirill-Simonovich — CHANGES_REQUESTED** [review]

File: src/test/java/com/stormnet/ChromeMainPageTest.java (hunk 1 above)
Comment (nataliashukalovich) [reply src/test/java/com/stormnet/ChromeMainPageTest.java]: hi @Kirill-Simonovich, I'm wondering what your comment 'open() use once per test' means

is it regarding not using method open() twice in the same test?

thx

**Comment by Kirill-Simonovich** [comment]
Everything you wanted to implement through locators

**Review by Kirill-Simonovich — APPROVED** [review]

File: src/test/java/com/stormnet/FFMainPageTest.java (hunk 3 above)
Comment (nataliashukalovich) [reply src/test/java/com/stormnet/FFMainPageTest.java]: @Kirill-Simonovich yes, it works. only 1 window opens

**Review by Kirill-Simonovich — APPROVED** [review]
Merge

**Pull request merged**